ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```diff
@@ -9,5 +9,10 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  env:
+  - name: COMPUTE_NODE_TYPE
```

Comment:
on Azure we are very limited to what instance types we can use due to quota limitations.. so I think this will cause a lot of users to trip into failures.. but since aws already supports this https://github.com/openshift/release/pull/14350/files maybe this is aceptable..